index.js

```diff
@@ -0,0 +1,239 @@
+const Twit = require("twit");
+const config = require("config");
+const winston = require("winston");
+require("winston-daily-rotate-file");
+const Client = require("bitcoin-core");
+const lbry = new Client({
+  version: "0.12.0",
+  username: config.get("lbrycrd.username"),
+  password: config.get("lbrycrd.password"),
+  port: config.get("lbrycrd.port")
+});
+const logger = winston.createLogger({
+  level: "info",
+  format: winston.format.json(),
+  transports: [
+    new winston.transports.DailyRotateFile({
+      filename: "tipbot-%DATE%.log",
+      dirname: "./logs",
+      datePattern: "YYYY-MM-DD-HH",
+      zippedArchive: true,
+      maxSize: "20m",
+      maxFiles: "14d"
+    }),
+    new winston.transports.Console({
+      format: winston.format.simple(),
+      level: "debug"
+    })
+  ]
+});
+
+const T = new Twit({
+  consumer_key: config.get("twitter.consumer_key"),
+  consumer_secret: config.get("twitter.consumer_secret"),
+  access_token: config.get("twitter.access_token"),
+  access_token_secret: config.get("twitter.access_token_secret"),
+  timeout_ms: 60 * 1000, // optional HTTP request timeout to apply to all requests.
+  strictSSL: true // optional - requires SSL certificates to be valid.
+});
+
+const stream = T.stream("statuses/filter", { track: config.get("bot.handle") });
+logger.info("Started LBRY twitter tipbot.");
+
+stream.on("tweet", function(tweet) {
+  if(tweet.user.screen_name === config.get("bot.handle").substring(1)) return;
+  let msg = checkTrunc(tweet);
+  msg = msg.slice(msg.indexOf(config.get("bot.handle"))).split(" ");
+  checkTweet(tweet, msg);
+});
+
+function checkTweet(tweet, msg) {
+  switch (msg[1]) {
```

Comment:
...thus if the split didn't go right, there is a chance that the bot crashes on this line as the array doesn't have at least 2 entries.

I would suggest adding a check somewhere in-between.

Reply:
Check added!